### ISSUE TITLE
New Apex Assert Methods should be allowed

### ISSUE THREAD

**Issue by Szandor72** [nawforce/apex-assist#33]
My team is very happy using Apex Assist and wants to Adopt the new [Apex Assert Methods](https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_class_System_Assert.htm) introduced recently. 

ApexAssist currently flags these with 'not found'.

```
    Assert.areEqual(
      [SELECT COUNT() FROM Task],
      (ACCOUNTS_NUMBER * TASKS_NUMBER * numberOfTaskTypes),
      'Failed to get expected amount of tasks'
    );
```

```
[{
	"resource": "/C:/Users/...",
	"owner": "apex-assist",
	"code": "ApexAssist​",
	"severity": 8,
	"message": "No variable or type found for 'Assert' on 'XYZTriggerHandlerTest'",
	"startLineNumber": 70,
	"startColumn": 5,
	"endLineNumber": 70,
	"endColumn": 11
}]
```

Would it be possible to add the new Assert Class to ApexAssist?

**Comment by chris-thejok3r** [comment]
That would be great 🥳

**Comment by nawforce** [comment]
Thanks for the input. We have support available for this but I have not yet released it. I will try push out an update soon.

**Comment by nawforce** [comment]
This is available now in the 2.0.0 release.
